### PR TITLE
Add Changes log

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -169,6 +169,22 @@ Only aggregate a specific repository using `fnmatch`_:
...
+Changes
+=======
+
+1.1.0 (Feb, 01, 2017)
+---------------------
+
+* Use setuptools_scm for the release process (https://github.com/acsone/git-aggregator/pull/10)
+* Expand env vars in config. (https://github.com/acsone/git-aggregator/pull/8)
+* Fetch only required remotes. (https://github.com/acsone/git-aggregator/pull/6)
+* Display readable error if config file not found. (https://github.com/acsone/git-aggregator/pull/2)
```
Comment (Yajo) [inline README.rst]: You forgot about `--depth` (#7) 😊

Comment (lmignon) [inline README.rst]: @Yajo Good catch. added.

**Comment by sbidoul** [comment]
@lmignon can you add a line to DEVELOP.rst to remind to update the changelog when releasing.

**Comment by lmignon** [comment]
@sbidoul done

**Pull request merged**